drupal/verify.sh

```diff
@@ -1,9 +1,10 @@
 #!/bin/sh
-
-set -e
-
+#
+# Starts the Acquia CMS site in an ephemeral container and confirms that it's
+# serving JSON:API data.
+#
 docker run --detach --publish 8080:80 --name drupal --rm phenaproxima/acquia_cms:headless
-docker cp settings.verify.php drupal:/opt/drupal/web/sites/default/settings.local.php
-docker exec --interactive --tty --workdir /opt/drupal/web drupal php ./core/scripts/db-tools.php import ../database.php.gz
+# Give the container time to start Apache.
+sleep 2
 curl --fail --silent http://127.0.0.1:8080/jsonapi
```

Comment:
Outside of scope here but I noticed that with `--silent`,  this doesn't output anything on the console 😅